src/components/TimerControlButtons.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+const TimerControlButtons = ({props, onStartClick, onResetClick}) =>{
+
+    function handleStartClick(event){
+        onStartClick(event.target.name);
+    }
```

Comment:
Używaj arrow function, od powstania es6 minęło już 5 lat, w tych czasach jest to standard.
```suggestion
    const handleStartClick = (event) => onStartClick(event.target.name);
```

Jakbyś chciał żeby ten komponent miał wydajność równą komponentowi klasowemu to powinieneś użyć również hooka `useCallback`:
```suggestion
    const handleStartClick = React.useCallback((event) => onStartClick(event.target.name), [onStartClick]);
```
https://reactjs.org/docs/hooks-reference.html#usecallback